### ISSUE TITLE
Can't select correct active tab when list of dynamic tabs changes

### ISSUE THREAD

**Issue by bryanbak** [angular-ui/bootstrap#5805]
### Bug description:
I am using a tabset that has 1 static tab then loops through a list of dynamic tabs. The problem I'm running into is that some actions trigger a new set of tabs to be loaded and then set one of those new tabs as the active tab. When I do this the scope variable I'm using for the active tab index gets set to null and either no tab or the wrong tab gets selected.

Before upgrading from ubis 0.13.4 to 1.2.5 I accomplished this by setting the bool "active" property on each tab after they were reloaded. After the upgrade and now having active on the tabset pointing to an index I am unable to get this working. The linked plunker demonstrates this. I've tried a couple variations and am unable to get it working in every case which makes me think it's a bug.

### Link to minimally-working plunker that reproduces the issue:
http://plnkr.co/edit/vIt6EZI15s1Q2wGY7qkq

### Version of Angular, UIBS, and Bootstrap

Angular: 1.4.5

UIBS: 1.2.5

Bootstrap: 3.3.4

**Comment by deeg** [comment]
This is likely a timing/digest issue as wrapping the setting of activeTabIndex after updating tabs in a timeout fixes the issue.

[See this updated plunker](http://plnkr.co/edit/9rwbXX19rYZDsezdp0uG?p=preview)

I'm not sure this is something that can be fixed in our library since I think the issue is with ng-repeat not being finished rendering the new tabs into the dom yet?

@wesleycho, thoughts?

**Comment by icfantv** [comment]
@bryanbak, please be respetcful of our time by searching both open and closed issues prior to entering anything (also per the text you deleted when you entered this issue).  As explained in several other threads, we are not going to support dynamic tabs indices (and @deeg gave you a work around).

Closing as won't support.

**Comment by bryanbak** [comment]
@icfantv not trying to be disrespectful of anyones time. I'm sure I wasted much more of my time than any of yours :)
Apparently my search skills weren't strong this morning cause I searched the docs, stackoverflow, and open/closed bugs before posting this.
It might be good to clarify that dynamic tabs aren't supported on the docs page. I thought it would be fine since my example is very similar to the first example in the docs ng-repeating over a list of tabs. And this same thing was working prior to upgrading uibs.
